iothub/service/samples/readme.md

```diff
@@ -17,11 +17,11 @@ Service samples were moved to [Azure-Samples/azure-iot-samples-csharp][samples-r
 [samples-repo]: https://github.com/Azure-Samples/azure-iot-samples-csharp
 [service-samples]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/service
 [adm-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/service/AutomaticDeviceManagementSample
-[device-streaming-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/service/DeviceStreamingSample
```

Comment:
Thanks for catching this dead link!

We've actually moved our "preview" samples to the `preview` branch, so this should point to the following instead: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/preview/iot-hub/Samples/service/DeviceStreamingSample